task/FP-1308 Update README for server testing (#532)
TACC/Core-Portal: README.md

- `npm run lint:css -- --fix`

Server-side Python code is linted via Flake8, and is also enforced on commits to the repo. To see server side linting errors, run `git diff -U0 main | flake8 --diff` from the command line.

> This requires you to run on the Core Portal container:

Reviewer: ```suggestion
To do so, run the following in the `core_portal_django` container:
```

```

> This requires you to run on the Core Portal container:

Reviewer: ```suggestion
```
